single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -217,32 +217,36 @@ protected void drawVoltageLevel(VoltageLevelGraph graph,
                                     GraphMetadata metadata,
                                     LabelProvider initProvider,
                                     StyleProvider styleProvider) {
+        Element g = root.getOwnerDocument().createElement(GROUP);
+        g.setAttribute("id", graph.getId());
```

Comment:
You need to escape the id in case non-allowed characters are used

Reply:
Done